src/components/AddressForm.jsx

```diff
@@ -68,7 +59,7 @@ const AddressForm = (props) => {
         company
     } = address;
 
-    const title = isShipping ? "Editing shipping address" : "Editing billing address";
+    const title = "Editing shipping address";
```

Comment:
Super nit: It feels a bit odd having `Editing shipping address` in a component called `AddressForm`. My immediate thought when I find the `AddressForm` component in the tree would be to think it was a general purpose form that didn't imply shipping, billing or other.

It could mean changing the name of the component to `ShippingAddressForm` or adding some flexibility to the component so it can be used for any arbitrary `addressUse` (prop with a default of `shipping perhaps)